src/languages/en-US.ts

```diff
@@ -2492,6 +2492,28 @@ export default class extends Language {
 				Now get those coins flippin'.`,
 			examples: ['50 heads', '200 tails']
 		}),
+		COMMAND_HIGHERLOWER_DESCRIPTION: 'Play a game of Higher/Lower',
+		COMMAND_HIGHERLOWER_EXTENDED: builder.display('higherlower', {
+			extendedHelp: `Higher Lower is a game of luck. I pick 2 numbers, and you have to guess if the 2nd is **higher** or **lower** then the 1st, using the ⬆ or ⬇ emojis.
+			Your winnings increase as you progress through the rounds, and can cashout any time pressing the 💰 reaction.
+			Be warned tho! The further you go, the more chances you have to lose the winnings.`
```

Comment:
```suggestion
			Be warned though! The further you go, the higher the chance you have to lose everything!`
```

Reply:
I wanted to avoid `everything` since it sounded like it would drain their purse+vault

Reply:
hm fair point